alpenhorn/auto_import.py

```diff
@@ -197,6 +197,51 @@ def get_acqcorrinfo_keywords_from_h5(path):
     f.close()
     return {"integration": integration, "nfreq": n_freq, "nprod": n_prod}
 
+def get_acqhfbinfo_keywords_from_h5(path):
```

Comment:
This function is never called.

Presumably you want to call it before creating the new `HFBAcqInfo` entry below to get the data to populate the new record with.

Down here: https://github.com/chime-experiment/alpenhorn/blob/e4685ef8dd42040f49960edc8f84ba1ee8cce7dd/alpenhorn/auto_import.py#L536